docs/glossary.md

```diff
@@ -0,0 +1,203 @@
+*Last update: 2024/01/29*
+
+
+This glossary provides terms and definitions for Runtimes OpenJ9 &reg;.
+{:shortdesc}
+
+The following cross-references are used in this glossary:
+
+- *See* refers you from a nonpreferred term to the preferred term or from an abbreviation to the spelled-out form.
+- *See also* refers you to a related or contrasting term.
+
+
+<!--If you do not want letter links at the top of your glossary, delete the text between these comment tags.-->
+
+[A](#glossa)
+[B](#glossb)
+[C](#glossc)
+[D](#glossd)
+[E](#glosse)
+[F](#glossf)
+[G](#glossg)
+[H](#glossh)
+[I](#glossi)
+[J](#glossj)
+[K](#glossk)
+[L](#glossl)
+[M](#glossm)
+[N](#glossn)
+[O](#glosso)
+[P](#glossp)
+[Q](#glossq)
+[R](#glossr)
+[S](#glosss)
+[T](#glosst)
+[U](#glossu)
+[V](#glossv)
+[W](#glossw)
+[Z](#glossz)
+
+<!--end letter link tags-->
+
+## A
+{: #glossa}
+
+
+### AWT font subsystem
+{: #x10447151}
+
+The Abstract Window Toolkit (AWT) package in Java that is used to create graphical user interface applications. It contains classes that help to implement common windows-based tasks, such as adding scroll bars, buttons, list items, and text boxes.
+
+## C
+{: #glossc}
+
+
+### cryptographic acclerator
+{: #x10447156}
+
+The use of hardware to run cryptographic operations to improve performance.
+
+## D
+{: #glossd}
+
+
+### dump
+{: #x2021514}
+
+A capture of storage information at the time of an error.
+
+### dynamic logical partitioning
+{: #x10447168}
+
+Partitioning with the capability to logically attach and detach a managed system's resources to and from a logical partition's operating system without rebooting.
+
+## H
+{: #glossh}
+
+
+### heap dump
+{: #x2788665}
+
+A dump that shows the content of the Java heap.
+
+### heuristic
+{: #x2085115}
+
+A problem-solving method or algorithm used to derive a workable solution in within a reasonable time.
+
+### huge page
+{: #x4550512}
+
+In Linux, a block of memory that can be 2 MB or 1 GB in size. Using huge pages increases the kernel-level performance by creating fewer pages and therefore fewer mapping tables that the kernel has to load.
+
+## J
+{: #glossj}
+
+
+### Java dump
+{: #x8172135}
+
+A dump that contains information from the Java application, such as the operating environment, locks, threads, hooks, shared classes, and class loaders.
+
+## L
+{: #glossl}
+
+
+### large page
+{: #x10447162}
+
+In Windows, a block of memory that can be 2 MB or 1 GB in size.
+
+### logical partition (LP,LPAR)
+{: #x2002959}
+
+A function of an operating system that creates segments of resources that can be run on copies, or instances, of the operating system and associated applications.
+
+### LP
+{: #x2029334}
+See [logical partition](#x2002959).
+
+
+### LPAR
+{: #x2029338}
+See [logical partition](#x2002959).
+
+
+## O
+{: #glosso}
+
+
+### OpenSSL
+{: #x10447142}
+
+An open source version of the SSL and TLS security protocols, which provide encryption and server authentication over the internet.
```

Comment:
It's more than this. Copying from openssl.org it's also a "toolkit for general-purpose cryptography", which is the part of OpenSSL used by OpenJ9 with OpenJDK.